Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,271 @@
+    string [int] currentWishables = {};
+    string [int] futureWishables = {};
+    // My kingdom for polymorphic filter :|
+    foreach index, wishable in wishables {
+        if (!wishable.shouldDisplay) continue;
+
+        if (wishable.theItem == $item[short writ of habeas corpus]) {
+            print(__quest_state["Level 11 Hidden City"].finished);
+            print(wishable.shouldDisplay);
+        }
+        if (wishable.currentlyAccessible) {
+            currentWishables.listAppend(showWishable(wishable));
+        } else {
+            futureWishables.listAppend(showWishable(wishable));
+        }
+    }
+    string [int] allWishables = {};
+    allWishables.listAppendList(currentWishables);
+    allWishables.listAppendList(futureWishables);
+    return allWishables;
+}
+
+record MonkeySkill {
+    int fingerCount;
+    skill theSkill;
+    string description;
+};
+
+RegisterResourceGenerationFunction("IOTMCursedMonkeysPawGenerateResource");
+void IOTMCursedMonkeysPawGenerateResource(ChecklistEntry [int] resource_entries) {
+    if (!lookupItem("cursed monkey's paw").have()) return;
+
+    string url;
+    string [int] description;
+    url = "main.php?action=cmonk&pwd=" + my_hash() + "";
+    description.listAppend("Return to monke. Wish for items or effects:");
+
+    Wishable [int] inRunWishables = {
+        new Wishable(
+            $item[sonar-in-a-biscuit],
+            $effect[none],
+            "",
+            get_property("questL04Bat") != "finished" &&
+                !locationAvailable($location[The Boss Bat's Lair]),
+            locationAvailable($location[Guano Junction])
+        ),
+        new Wishable(
+            $item[stone wool],
+            $effect[none],
+            "",
+            !locationAvailable($location[The Hidden Park]) &&
+                available_amount($item[stone wool]) < 1,
+            locationAvailable($location[The Hidden Temple])
+        ),
+        new Wishable(
+            $item[amulet of extreme plot significance],
+            $effect[none],
+            "",
+            !locationAvailable($location[The Castle In The Clouds In The Sky (Basement)]),
+            locationAvailable($location[The Penultimate Fantasy Airship])
+        ),
+        new Wishable(
+            $item[mohawk wig],
+            $effect[none],
+            "",
+            !locationAvailable($location[The Castle In The Clouds In The Sky (Basement)]),
+            locationAvailable($location[The Penultimate Fantasy Airship])
+        ),
+        new Wishable(
+            $item[soft green echo eyedrop antidote],
+            $effect[none],
+            "",
+            !locationAvailable($location[The Castle In The Clouds In The Sky (Basement)]),
+            locationAvailable($location[The Penultimate Fantasy Airship])
+        ),
+        new Wishable(
+            $item[book of matches],
+            $effect[none],
+            "",
+            my_ascensions() > get_property_int("hiddenTavernUnlock") &&
```

Comment:
the mafia wiki says inequality is the "tavern not unlocked" condition, not >; can we just swap this to a not-equal